src/TimeManager.jl

```diff
@@ -0,0 +1,52 @@
+#=
+    TimeManager
+
+This module facilitates calendar functions and temporal interpolations
+of data.
+=#
+
+module TimeManager
+
+using ..Utilities
+using Dates
+
+export current_date, strdate_to_datetime, datetime_to_strdate
+
+
+"""
+    current_date(cs, t)
+
+Return the model date at the current timestep
+
+# Arguments
+- `cs`: [CoupledSimulation] or [CouplerSimulation] containing info about the simulation
+- `t`: [Integer] number of seconds since simulation began
+"""
+current_date(cs, t) = cs.dates.date0[1] + Dates.Second(t)
```

Comment:
Please specify the argument types here.

Reply:
Is there a better way I should specify the types than `[CoupledSimulation] or [CouplerSimulation]` and `[Integer]`? I wasn't sure how to be more specific since there are multiple valid types.